File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    divisible = False
    result = ""
    if number % 3 == 0:
        divisible = True
        result += "Pling"
    if number % 5 == 0:
        divisible = True
        result += "Plang"
    if number % 7 == 0:
        divisible = True
        result += "Plong"
    if divisible:
        return result
    else:
        return str(number) 

File: solutions/typescript/raindrops/1/raindrops.ts
export function convert(plingPlanger: number): string {
  let result = "";
  if (plingPlanger % 3 === 0) {
    result += "Pling";
  }
  if (plingPlanger % 5 === 0) {
    result += "Plang";
  }
  if (plingPlanger % 7 === 0) {
    result += "Plong";
  }

  if (result === "") {
    return `${plingPlanger}`;
  } else {
    return result;
  }
}
